### PR TITLE
filtered tag remove empty name at edit query

### PR DESCRIPTION
**Pull request by taminif**
## What type of PR is this? (check all applicable)
- [x] Bug Fix

## Description
Filtered tag remove empty tag name at edit query on backend.

## Related Tickets & Documents
#3734
`2. We should change the backend to filter them as well.`

### CODE REVIEW

File: redash/handlers/queries.py [hunk 1]
```diff
@@ -332,6 +332,9 @@ def post(self, query_id):
         if 'query' in query_def:
             query_def['query_text'] = query_def.pop('query')
 
+        if 'tags' in query_def:
+            query_def['tags'] = [tag for tag in query_def['tags'] if tag]
```
Comment (rauchy) [inline redash/handlers/queries.py]:
```suggestion
            query_def['tags'] = filter(None, query_def['tags'])
```

Comment (taminif) [reply redash/handlers/queries.py]: @rauchy Thanks suggestions!

**Comment by taminif** [comment]
@rauchy Please review once again!

**Comment by arikfr** [comment]
Thanks, @taminif !

**Pull request merged**